jobrunner/queries.py

```diff
@@ -24,3 +25,24 @@ def calculate_workspace_state(workspace):
 
 def group_by(iterable, key):
     return groupby(sorted(iterable, key=key), key=key)
+
+
+def get_flag(name):
+    """Get the current value of a flag, None if not set."""
+    # Note: fail gracefully if the flags table does not exist
+    # This means we don't need to worry about it in local_run.
+    try:
+        return find_one(Flag, id=name).value
+    except ValueError:
```

Comment:
I should have made `find_one` raise something more specific here, something like `DoesNotExist` and `MultipleObjectsReturned` like Django does. Then we could catch exactly that error. I guess given the primary key constraint on ID we're never going to get multiple records back here, but I worry that `ValueError` is still quite broad. If you feel like adding another exception class then great, but I don't think essential.

Reply:
I'll leave it for now, enough yaks already!